redis/commands.py

```diff
@@ -504,6 +504,13 @@ def lastsave(self):
         """
         return self.execute_command('LASTSAVE')
 
+    def lolwut(self, *version_numbers):
+        "Get the Redis version and a piece of generative computer art"
+        if version_numbers:
```

Comment:
```suggestion
       return self.execute_command('LOLWUT VERSION', *version_numbers)
```

If version numbers is empty, it'll simply be concatenated as empty in the list. There's no need to use the if check.

Reply:
I believe we need the `if` condition. If `version_numbers` is an empty list, then we don't want to include the string `VERSION` in the command. But if `version_numbers` contains values, then we want to include `VERSION` in the command.

If we pass an empty list along with `LOLWUT VERSION`, we get this error:

```python
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/rajiv.shah/Documents/Code/pottery/pottery/monkey.py", line 86, in lolwut
    return self.execute_command('LOLWUT VERSION', *version_numbers)
  File "/Users/rajiv.shah/Documents/Code/pottery/venv/lib/python3.9/site-packages/redis/client.py", line 901, in execute_command
    return self.parse_response(conn, command_name, **options)
  File "/Users/rajiv.shah/Documents/Code/pottery/venv/lib/python3.9/site-packages/redis/client.py", line 915, in parse_response
    response = connection.read_response()
  File "/Users/rajiv.shah/Documents/Code/pottery/venv/lib/python3.9/site-packages/redis/connection.py", line 756, in read_response
    raise response
redis.exceptions.ResponseError: value is not an integer or out of range
```